es/lessons/basics/strings.md

```diff
@@ -0,0 +1,158 @@
+---
+version: 1.1.1
+title: Cadenas
+---
+
+Cadenas, listas de caracteres, Grafemas y puntos de código.
+
+{% include toc.html %}
+
+## Cadenas
+
+Las cadenas en Elixir no son mas que una secuencia de bytes. Vamos a ver un ejemplo:
+
+```elixir
+iex> string = <<104,101,108,108,111>>
+"hello"
+iex> string <> <<0>>
+<<104, 101, 108, 108, 111, 0>>
+```
+
+Concatenando la cadena con el byte `0` IEx muestra la cadena como un binario porque ya no es una cadena válida. Este truco puede ayudarnos a ver los bytes subyacentes de cualquier cadena.
+
+>NOTA: Usando la sintaxis << >> estamos diciéndole al compilador que los elementos dentro de esos símbolos son bytes.
+
+## Listas de caracteres
+
+Internamente las cadenas en Elixir son representadas como una secuencia de bytes en lugar de un arreglo de caracteres. Elixir también tiene un tipo de lista de caracteres. Las cadenas de Elixir están encerradas con comillas dobles mientras que las listas de caracteres están encerradas con comillas simples.
+
+¿Cuál es la diferencia? Cada valor en una lista de caracteres es el punto de código Unicode de un carácter mientras que en un binario los puntos de código se codifican en UTF-8. Vamos a profundizar en:
+
+```elixir
+iex(5)> 'hełło'
+[104, 101, 322, 322, 111]
+iex(6)> "hełło" <> <<0>>
+<<104, 101, 197, 130, 197, 130, 111, 0>>
+```
+
+`322` es el punto de código Unicode para ł pero esta codificado en UTF-8 como los 2 bytes `197` y `130`.
+
+Cuando programamos en Elixir usualmente usamos cadenas no listas de caracteres. El soporte para listas de caracteres esta incluido principalmente porque es requerido para algunos módulos Erlang.
+
+Para mas información podemos ver la [`Guía de inicio`](http://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html).
+
+## Grafemas y puntos de código
+
+Los puntos de código son solo caracteres Unicode los cuales están representados por uno o mas bytes dependiendo de la codificación UTF-8. Los caracteres fuera del conjunto de caracteres US ASCII siempre se codificarán con mas de un byte. Por ejemplo los caracteres latinos con tilde o acento (`á, ñ, è`) son usualmente codificados con dos bytes. Los caracteres de idiomas asiáticos son frecuentemente codificados con tres o cuatro bytes. Los grafemas consisten de múltiples puntos de código que son renderizados como un simple carácter.
```

Comment:
I think it's better to use _dibujados_ instead of _renderizados_ here.

Reply:
Yeah. I just realized "renderizados" doesn't exist in spanish 😅 

It's fixed now